butterknife-compiler/build.gradle

```diff
@@ -13,14 +13,14 @@ for (File file : sdkHandler.sdkLoader.repositories) {
 }
 
 dependencies {
-  compile project(':butterknife-annotations')
-  compile deps.auto.common
-  compile deps.javapoet
+  implementation project(':butterknife-annotations')
+  implementation deps.auto.common
+  api deps.javapoet
```

Comment:
This is because of how we're running the compiler in the tests. Just going to leave it for simplicity.